app/controllers/thesis_controller.rb

```diff
@@ -6,6 +6,7 @@ class ThesisController < ApplicationController
 
   def new
     @thesis = Thesis.new
+    @thesis.advisors = [Advisor.new({name: ''})]
```

Comment:
Hmmm. We should probably use the syntax from `edit` that works here as well for consistency.